### DISCUSSION TITLE
not able to do the first step even, what s this 🥲

### DISCUSSION THREAD

**Discussion by suprith-hub** [Unleash/unleash#6064] [General]
(base) PS E:\G> git clone git@github.com:Unleash/unleash.git
Cloning into 'unleash'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by sighphyre** [comment]
Hey @Era-cell,

Oh sorry this is happening!

I think this is likely an issue with Github itself, those look like the setup steps from the main Unleash repo, that assumes you already have an [SSH key setup](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account), which I think is the problem here.

If you're just looking to experiment a little and run the docker compose, you can always clone with the https URL: https://github.com/Unleash/unleash.git

**Reply by suprith-hub** [reply]
Yeah I just resolved it, Im happy that you are replying

**Reply by sighphyre** [reply]
Glad that got you moving! And no worries!
